src/DocRaptor.php

```diff
@@ -71,13 +74,14 @@ public function setDocumentUrl($document_url)
      */
     public function setDocumentType($document_type)
     {
+        $document_type_filtered = strtolower(trim($document_type));
         $allowedValues = array('xls', 'xlsx', 'pdf');
 
-        if (! in_array(strtolower(trim($document_type)), $allowedValues)) {
+        if (! in_array($document_type_filtered, $allowedValues)) {
             throw new Exception('Value not accepted by method.');
```

Comment:
Perhaps the message could be more clear like "Document Type must be in {xls, xlsx, pdf}". Not a blocker.